tools/topology/sof-apl-pcm512x.m4

```diff
@@ -43,14 +43,14 @@ dnl     time_domain, sched_comp)
 PIPELINE_PCM_ADD(sof/pipe-low-latency-playback.m4,
 	1, 0, 2, s32le,
 	1000, 0, 0,
-	48000, 48000, 48000)
+	44100, 44100, 44100)
```

Comment:
The build could make create topology versions of this with 44.1 kHz and 48 kHz but people (me) who may need 48 kHz can do it later.

Maybe this topology version could be called sof-apl-pcm512x-master-44k1.tplg to call out the difference to other SOF topologies.

Reply:
I made this configurable with a macro, so you can just take the result and copy it on the target.